src/pykeen/nn/functional.py

```diff
@@ -1186,3 +1189,82 @@ def cross_e_interaction(
         x = dropout(x)
     # similarity
     return (x * t).sum(dim=-1)
+
+
+def boxe_interaction(
+    # head
+    h_pos: torch.FloatTensor,
+    h_bump: torch.FloatTensor,
+    # relation box: head
+    rh_base: torch.FloatTensor,
+    rh_delta: torch.FloatTensor,
+    rh_size: torch.FloatTensor,
+    # relation box: tail
+    rt_base: torch.FloatTensor,
+    rt_delta: torch.FloatTensor,
+    rt_size: torch.FloatTensor,
+    # tail
+    t_pos: torch.FloatTensor,
+    t_bump: torch.FloatTensor,
+    # power norm
+    tanh_map: bool = False,
```

Comment:
6. and thus arrive here :confused:

Reply:
I don't know, really. All I can say, very confidently, is that the model works when I replace this False with True.

Reply:
This should be changed, anyway, but it suggests a problem if the default is being used in the first place.